### PR TITLE
Created new Django project under tray folder for Dev Sessions Project

### PR DESCRIPTION
**Pull request by traytorous**

### CODE REVIEW

File: tray/apicontest/@
Comment (chynasan) [inline tray/apicontest/@:1]: Is this a duplicate file?

**Review by chynasan — COMMENTED** [review]
This looks good other than the potential duplicate file!

File: tray/apicontest/@
Comment (traytorous) [reply tray/apicontest/@:1]: @chynasan I fixed the issue

**Comment by epacific1** [comment]
LGTM

**Pull request merged**